.circleci/config.yml

```diff
@@ -67,6 +78,7 @@ workflows:
             - install
       - build:
           requires:
+            - install
```

Comment:
What does requiring `install` for `build` change?

Reply:
Nothing, I guess it's a _"just-in-case"_. But `install` should be blocked by the `test` job anyway which requires `install` there